spark/src/test/scala/org/apache/spark/sql/delta/columnmapping/DropColumnMappingFeatureSuite.scala

```diff
@@ -82,17 +85,12 @@ class DropColumnMappingFeatureSuite extends RemoveColumnMappingSuiteUtils {
          |USING delta
          |TBLPROPERTIES ('${DeltaConfigs.COLUMN_MAPPING_MODE.key}' = 'name',
          |        '${DeltaConfigs.ENABLE_DELETION_VECTORS_CREATION.key}' = 'false',
-         |        'delta.minReaderVersion' = '1',
-         |        'delta.minWriterVersion' = '1')
+         |        'delta.minReaderVersion' = '3',
+         |        'delta.minWriterVersion' = '7')
```

Comment:
Why did we need to update this test? the minReaderVersion and minWriterVersion are "suggestions" and are ignored anyways since we enable DVs which will make this table have (3,7) in the end anyways. Are we enforcing users' minReaderVersion and minWriterVersion after this change?